src/main/java/util/constants/Urls.java

```diff
@@ -0,0 +1,8 @@
+package util.constants;
+
+public class Urls {
```

Comment:
[Optional] could be enum or refactored to store base url as constant and endpoints as parameter of page object's annotations